install/tools/setupMeshEx.sh

```diff
@@ -122,17 +110,16 @@ function istio_provision_certs() {
   if [[ -n "$NS" ]] ; then
     NS="-n $NS"
   fi
+
+  # shellcheck disable=SC2086
+  go run ./security/tools/generate_cert -client \
```

Comment:
Do users run this from the release? If so this won't work, since the code is not part of the release

Reply:
Right - luckily "go run istio.io/istio/...' is now working, and seems reasonable for an 'example tool' - will add a comment that users should use a proper cert generator.

Reply:
Will update this in separate PR - not clear if adding this to istioclt proper is feasible.